### ISSUE TITLE
Git submodules are not updated

### ISSUE THREAD

**Issue by 0dysseas** [dependabot/dependabot-core#4034]
I have created a private, test repo (i.e. `test-superproject`) that contains just a submodule which is another private repo. The `test-superproject` has the following `.yaml` (this is present only in the default, `dev`, branch- not on `main`):

**dependabot.yml content**
```
version: 2
updates:
  - package-ecosystem: "gitsubmodule"
    directory: "/"
    schedule:
      interval: "daily"
    assignees:
      - "0dysseas"
    labels:
      - "submodule-updates"
    # Must use the full team name, including the organization, as if you were @mentioning the team
    reviewers:
      - "0dysseas"
    # Allow dependabot to open up to 10 open pull requests. Default is 5.
    # open-pull-requests-limit: 10
```

**Package ecosystem**
`gitsubmodule`

**What you expected to see, versus what you actually saw**
In case there are submodule updates, I expect a new PR on `~05:00 UTC`. If there are no updates I expect to see that the submodule has been checked for any new updates on `~05:00 UTC`.

I saw neither of those. In fact, dependabot ran only once, when I manually changes its `.yml` entry.

**Images of the diff or a link to the PR, issue or logs**

![image](https://user-images.githubusercontent.com/31179964/125193710-8a487000-e256-11eb-940e-6279fcd63cb4.png)

Am I missing something? Have I configured something wrong? Any help will be greatly appreciated.

**Comment by 0dysseas** [comment]
I removed the `dependabot-preview` app from my account and dependabot ran today at ~14:10 UTC time. Don't know f if this was correlated but the automatic submodules update now works. I'm closing the issue.
